src/binder/bind/bind_copy.cpp

```diff
@@ -194,16 +185,16 @@ static bool skipPropertyInFile(const Property& property) {
 
 expression_vector Binder::bindExpectedNodeFileColumns(
     TableSchema* tableSchema, ReaderConfig& readerConfig) {
-    expression_vector columns;
+    // Resolve expected columns.
```

Comment:
I think we should make the expected column names case insensitive

Reply:
Mildly I agree, though we could move this to a future PR (file an issue for it).

Reply:
This one doesn't matter, because we don't expose column name to user for `COPY` statement. For `LOAD` this is probably true. I'll make a change after the release.